R/Functions_L1_Pop.R

```diff
@@ -437,9 +438,10 @@ createCastePop <- function(x, caste = NULL, nInd = NULL,
     }
     ret@sex[] <- "F"
     simParamBee$changeCaste(id = ret@id, caste = "virginQueens")
-    if (!is.null(year)) {
-      ret <- setQueensYearOfBirth(x = ret, year = year, simParamBee = simParamBee)
-    }
+
+    # if (!is.null(year)) {
```

Comment:
How do we address this instead of this code? Did you remove it because of parallelisation clash?